kotest-assertions/kotest-assertions-shared/src/jvmMain/kotlin/io/kotest/assertions/async/timeout.jvm.kt

```diff
@@ -9,7 +9,7 @@ import kotlin.time.toKotlinDuration
  * Assert [thunk] does not complete within [duration].
  */
 @Deprecated(
-   "Updated to use Kotlin time types. This function will be removed in 6.0.",
+   "Updated to use Kotlin time types. This function will be removed in 6.1.",
```

Comment:
But the 6.0 guys get a kick to 6.1.

Poor 7.0 deprecations :(

Reply:
Ha. I was basing it on time deprecated